adapterbox/install.sh

```diff
@@ -0,0 +1,10 @@
+sudo apt-get --yes install libcurl3 php7.0-curl php7.0-cli php7.0-xml
```

Comment:
7.0 is more than old. I think a simply only put 7 instead of 7.0 would be enough

Reply:
I would do that then fixing the whole script. The scope now is just to make everything available as it is.